test/unit/services/igdbClient.spec.ts

```diff
@@ -0,0 +1,72 @@
+import * as igdbClient from '../../../src/services/igdbClient';
```

Comment:
@julian3ng should the linter also handle import paths? This one looks especially yikes so just wondering

Reply:
created https://gaming-backlog.atlassian.net/browse/GB-30